src/update-feeds/check-feeds.ts

```diff
@@ -0,0 +1,164 @@
+import { go } from '@api3/promise-utils';
+import { ethers } from 'ethers';
+
+import { calculateMedian, checkUpdateConditions } from '../condition-check';
+import { logger } from '../logger';
+import { getStoreDataPoint } from '../signed-data-store';
+import { getState } from '../state';
+import type { ChainId, ProviderName } from '../types';
+import { multiplyBigNumber } from '../utils';
+
+import { getApi3ServerV1 } from './api3-server-v1';
+import type { ReadDapiWithIndexResponse } from './dapi-data-registry';
+import { decodeBeaconValue } from './update-feeds';
+import type { UpdatableDapi } from './update-transactions';
+
+export const checkFeeds = async (
```

Comment:
Why change the name from `getUpdatableFeeds`?

Reply:
ah, it doesn't bother me, I think this was the name of the original parent function. I'll revert it to `getUpdatableFeeds` 👌